docs/commands/list.md

```diff
@@ -0,0 +1,113 @@
+# `list` command
+
+The `list` command is used to list all the entries in the password store or at a given prefix.
+
+## Synopsis
+
+```bash
+$ gopass ls
+$ gopass ls path/to/entries
+```
+
+- List all the entries in the password store including the one in mounted stores: `gopass list`
+- List all the entries in a given folder showing their relative path from the root: `gopass list path/to/entries`
+
+Note: `list` will not change anything, nor encrypt or decrypt anything.
+
+## Flags
+
+Flag | Aliases | Description
+---- | ------- | -----------
+`--limit value` | `-l value`| Max tree depth (default: -1)
+` --flat `      |` -f`      | Print flat list (default: false)
+` --folders`    | `--fo`    |  Print flat list of folders (default: false)
```

Comment:
`--fo` is a strange shortcut. Maybe we should rename this to `-d`?

Reply:
LGTM, at first I wanted to go with `-ff` "flat folder" and ended up with `-fo` ^^'
I'll change it in another PR.

Reply:
`-d` for directory make sense to me.

Reply:
done